### PR TITLE
Improve doc

### PR DESCRIPTION
**Pull request by huyvohcmc**
Improve README's typo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,17 +31,20 @@ README. Note that vim8 support is simply a bonus. It's not the goal of ncm2.
     " ncm2 requires nvim-yarp
     Plug 'roxma/nvim-yarp'
 
-    " enable ncm2 for all buffer
-    autocmd BufEnter * call ncm2#enable_for_buffer()
+    " enable ncm2 for all buffers
+    augroup EnableNcm2ForAllBuffers
+      autocmd!
+      autocmd BufEnter * call ncm2#enable_for_buffer()
+    augroup
```
Comment (roxma) [inline README.md]: I don't think `augroup` is needed here.

Comment (huyvohcmc) [reply README.md]: Why? When you source your vimrc for the second time the autocommand will be added again, efficiently calling `ncm2#enable_for_buffer()` twice when opening a file.

Comment (roxma) [reply README.md]: Why would anyone source the vimrc twice? vimrc is only for startup-time configuration. Multiple-sourcing makes things complicated and messy.

Comment (huyvohcmc) [reply README.md]: Understandable, I removed it.

**Comment by roxma** [comment]
Thanks

**Pull request merged**